DataDictionary.md

```diff
@@ -41,5 +42,13 @@
 * The database also features three year compliance history by quarter
 
 ## Inspection
-
+* ECHO database features number of inspections within 5 years, including the dates of the inspections
+* There are lag times between the date of the inspection and the time the data is posted, sometimes spanning months
+* Inspections, information gathering, and self-reported data spurs warning letters or notice of violation
+* Smaller facilities may receive less frequeny inspections
+
+## Quarter
+* Defines a unit of time of approximately three months to define the quanity of noncompliance or violations within the time period
```

Comment:
potential alt working as define is used twice: "A quarter is a unit of time that spans three months and is the standard base unit for reporting noncompliance events or violations in ECHO. For the Clean Water Act, the permittee will generally report three times within the quarter."